indexing-service/src/main/java/org/apache/druid/indexing/common/config/TaskConfig.java

```diff
@@ -118,18 +121,10 @@
   @JsonProperty
   private final boolean encapsulatedTask;
 
-  @Deprecated
-  @JsonProperty("baseTaskDir")
-  private final String baseTaskDirPath;
-
-  // Use multiple base files for tasks instead of a single one
-  @JsonProperty
-  private final List<String> baseTaskDirPaths;
-
   @JsonCreator
   public TaskConfig(
       @JsonProperty("baseDir") String baseDir,
-      @Deprecated @JsonProperty("baseTaskDir") String baseTaskDirPath,
+      @JsonProperty("baseTaskDir") String baseTaskDir,
```

Comment:
should this still be deprecated as a json config for configuring task locations?

Reply:
Okay, just fixed this by setting the config from the ForkingTaskRunner and using a different build method from the ThreadingTaskRunner.